docs/mastering-plone/volto_overrides.md

```diff
@@ -402,7 +402,7 @@ The result should look like this:
 - You need to restart Volto when you add a new override.
 
 ```{seealso}
-- <https://training.plone.org/5/volto/override-components.html>
-- <https://training.plone.org/5/voltohandson/header.html#header-component>
-- <https://training.plone.org/5/volto/override-views.html>
+- {doc}`volto/override-components`
+- {ref}`header-component`
```

Comment:
This did not render to a link because it lacks a label on the target.

https://github.com/plone/training/blame/main/docs/voltohandson/header.md#L109

Would you please add a label there, and then update the target here?